backend/api/views.py

```
@@ -1,3 +1,96 @@
-# Create your views here.
+class RoleViewSet(viewsets.ModelViewSet):
+    queryset = Role.objects.all()
+    serializer_class = RoleSerializer
+    # Removed permission_classes
+
+class CustomUserViewSet(viewsets.ModelViewSet):
+    queryset = CustomUser.objects.all()
+    serializer_class = CustomUserSerializer
+    
+    # Removed permission_classes
+    def get_serializer_class(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_serializer_class`

(ANN201)